katas/adriel-gimenes/02-06/Title-Case/TitleCase.cs

```diff
@@ -0,0 +1,23 @@
+namespace katas.AdrielGimenes;
+
+public class TitleCase
+{
+    public static string TitleCases(string title, string minorWords = "")
+    {
+        minorWords = minorWords == null ? "" : minorWords;
```

Comment:
minorWords will never be null if it is and optional argument set to ""

Reply:
Actually even with the default value for `minorWords` in place, if we remove line `7` here, if we pass `null` as `minorWords`, like the `Test3()` does, it will fail.

Reply:
You're right!